elastic/kibana
# [maps] fix icon markers fail to load when browser zoomed out

Fixes https://github.com/elastic/kibana/issues/134131

To test
* load web logs sample data
* Create new map and add documents layer
* Style markers as icons.
* Save map
* Zoom out browser until its 50% or less.
* reload page. Icons should still be displayed
